corehq/apps/hqwebapp/session_details_endpoint/views.py

```diff
@@ -41,14 +43,20 @@ def post(self, request, *args, **kwargs):
         if not session_id:
             return HttpResponseBadRequest()
 
-        user = get_django_user_from_session_key(session_id)
+        session = get_session(session_id)
```

Comment:
This will return `None` when the session has expired which will raise the 404 later